deploy/kubernetes/releases/csi-digitalocean-v1.1.0.yaml

```diff
@@ -381,6 +386,7 @@ spec:
           image: digitalocean/do-csi-plugin:v1.1.0
           args :
             - "--endpoint=$(CSI_ENDPOINT)"
+            - "--token=$(DIGITALOCEAN_ACCESS_TOKEN)"
```

Comment:
This is not needed anymore. The node plugin driver doesn't use the token anymore.

Reply:
@fatih I've removed that line from the pull request.